spec/factories/teams.rb

```diff
@@ -17,5 +17,23 @@
     kind_of_sport "Football"
     description "This team is awesome"
     private false
+
+    after(:build) do |team|
+      team.owners = build_list :user, 1
+      team.members = team.owners
+    end
+  end
+
+  trait :with_two_owners do
+    after(:build) do |team|
+      team.owners = build_list :user, 2
+      team.members = team.owners
+    end
+  end
+
+  trait :with_five_members do
+    after(:build) do |team|
+      team.members = build_list :user, 5
```

Comment:
I think that the trait `:with_five_members` constructs an invalid Team model: It has one owner that **is not a member**. The reason for that is that the team's `members` (that contain the team owner after line 23) are **replaced** by the newly built list of team members in line 36. According to the (implicit?) acceptance criteria that every team owner is also a team member, the resulting Team model is invalid to my mind. 

One solution would be to append the newly built list of team members to the existing team members in line 36:

```ruby
team.members = team.members + build_list :user, 5
```

This would however require to change the "should be able to have multiple team members" test to something like:

```ruby
expect(team.team_members).to have_at_least(5).items
```

Reply:
Absolutely correct, I did not see this. Thanks for the hint! 👍